test/libsolidity/syntaxTests/experimental/parsing_stdlib_import_4.sol

```diff
@@ -0,0 +1,6 @@
+pragma experimental solidity;
+
+import * as stub from std.stub;
+// ----
+// Warning 2264: (std.stub:63-92): Experimental features are turned on. Do not use experimental features on live deployments.
```

Comment:
Yeah, continuing our discussion from the chat: I'd say this is perfectly fine for now. We can still consider filtering these warnings out (maybe even where they're emitted and not merely in the tests here), but for this PR, this we can keep this just as is.

Reply:
I mean, our end goal is to not have any warning be emitted from the stdlib implementation files, so if we implement them properly, this should take care of itself, no?

Reply:
I meant when stdlib is not longer experimental.